go-app-ussd_tb_check.js

```diff
@@ -957,6 +961,10 @@ go.app = function () {
       var answers = self.im.user.answers;
       var arm = answers.group_arm;
       var consent = answers.state_research_consent || answers.research_consent;
+
+      console.log("************************************************");
+      console.log("Group_arm" + arm);
+      console.log("Consent" + arm);
```

Comment:
Remove this statements

Reply:
@Buhle79 It's intentional for testing. I need to see what those values are. Remember the one we talked about.